style.css

```diff
@@ -0,0 +1,593 @@
+  background-color: #f7f7f9;
+}
+
+.container-3 {
+  background-color: white;
+  border-top-right-radius: 100px;
+  padding-left: 30px;
+  padding-top: 100px;
+  padding-right: 20px;
+  margin-bottom: 195px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.about {
+  font-size: 2.4em;
+  padding-bottom: 10px;
+}
+
+.about-div p {
+  padding-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-size: 16px;
+}
+
+.m-b {
+  margin-top: 20px;
+}
+
+.languages {
+  padding-top: 50px;
+}
+
+.languages-div {
+  display: flex;
+  align-items: center;
+  padding-left: 20px;
+  list-style: none;
+}
+
+.framework {
+  border-bottom: 1.5px solid #dfe1e6;
+  padding: 25px 0;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 60px;
+}
+
+.skills {
+  background-color: #f7f7f9;
+  margin-top: 20px;
+  padding-bottom: 15px;
+  padding-top: 15px;
+  border-radius: 10px;
+  margin-right: 20px;
+  display: flex;
+}
+
+.m-l {
+  margin-left: 16px;
+}
+
+.language {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 60px;
+}
+
+.section-2-desktop {
+  display: none;
+}
+
+.section-4 {
+  background-color: #6070ff;
+  background-image: url("Media/contact\ form\ shapes-mobile@2x.png");
+  background-position: right 0;
+  background-repeat: no-repeat;
+  width: 100%;
+  border-top-left-radius: 100px;
+}
+
+.section-4-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.full-name {
+  margin-bottom: 2rem;
+}
+
+.contact {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  margin-top: 80px;
+  margin-bottom: 10px;
+}
+
+.section-4-container p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 20px;
+  margin-top: 10px;
+  padding: 0 20px;
+}
+
+form {
+  width: 100%;
+  padding: 0 20px;
+}
+
+form input,
+textarea {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 15px 15px;
+}
+
+input::placeholder {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+}
+
+textarea::placeholder {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+}
+
+.submit-container {
+  display: flex;
+  align-items: flex-start;
+}
+
+.submit-btn {
+  padding: 12px 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: #fff;
+  border: none;
+  color: #6070ff;
+  text-decoration: none;
+  border-radius: 10px;
+  margin-right: -20px;
+}
```

Comment:
Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file.

## Solution:--

```
.btn:hover{
    background-color: #6070FF;
    color: white;
}
```

```
.btn:active{
    background-color: #2230D2;
    color: white;
}
```

## Demonstration:--

![image](https://user-images.githubusercontent.com/94223367/219416172-4958d9ca-1fa4-4068-adbb-ef3657bc7b3f.png)